docs/accessibility.md

```diff
@@ -0,0 +1,9 @@
+# Accessibility
+
+The **Microsoft MakeCode** team wants to be sure that every person on the planet can play with it to achieve more. It is very important for us when we're working to make computer science more accessible to consumers with disabilities.
```

Comment:
Not sure what to say here. This is kind of a policy statement. Maybe ask @Jaqster about what to say here.

Reply:
"This page describes the accessibility features available in MakeCode-based editors."

Reply:
+1 Peli's suggestion - keep it simple/factual.  The current wording sounds a bit arrogant.